esp8266-fastled-webserver/include/simplehacks/integer_seq.h

```diff
@@ -0,0 +1,150 @@
+/**
```

Comment:
SIMPLE FILE REVIEW

Adds non-STL, `constexpr` compatible integer sequences, using template meta programming.  Useful to statically initialize arrays where the size is not known until compilation time, for example.